### PR TITLE
chore(Android): Update Android versions

### PR DESCRIPTION
**Pull request by tylerjroach**
_Issue #, if available:_

_Description of changes:_
Updating Amplify Android versions to the latest.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by tylerjroach** [comment]
Will merge once approvals are given and release has propagated to Maven.

**Review by eeatonaws — APPROVED** [review]

**Pull request merged**
